resources/lang/en/patrols/new_cat.json

```
@@ -4327,5 +4327,393 @@
+                        "no_results": true
+                    }
+                ],
+                "history_text": {
+                    "scar": "m_c was scarred after foolishly trying to take on a c_n apprentice.",
+                    "death": "m_c died from injuries sustained after foolishly trying to take on a c_n apprentice."
+                },
+                "outsider_rep": -1
+            },
+            {
+                "text": "p_l hisses out a warning but, surprisingly, the kittypet refuses to leave without a fight. <i>Un</i>surprisingly, {PRONOUN/n_c:0/subject}{VERB/n_c:0/'re/'s} quickly put in {PRONOUN/n_c:0/poss} place, with p_l <i>easily</i> winning the ensuing scuffle.",
+                "exp": 10,
+                "weight": 5,
+                "new_cat": [
+                    ["meeting", "kittypet", "age:adolescent"]
+                ],
+                "relationships": [
+                    {
+                        "cats_to": ["n_c:0"],
+                        "cats_from": ["p_l"],
+                        "mutual": true,
+                        "values": ["dislike"],
+                        "amount": 10
+                    }
+                ],
+                "injury": [
+                    {
+                        "cats": ["n_c:0"],
+                        "injuries": ["claw-wound", "torn pelt", "torn ear", "minor_injury"],
+                        "scars": ["ONE", "TWO", "THREE", "FOUR", "RIGHTEAR", "LEFTEAR", "RIGHTBLIND", "LEFTBLIND", "BOTHBLIND", "MANLEG", "MANTAIL", "CATBITE", "CATBITETWO", "BRIDGE", "FACE", "SNOUT", "CHEEK", "THROAT", "TAILBASE", "TAILSCAR", "BACK", "SCRATCHSIDE", "SIDE", "BELLY"],
+                        "no_results": true
+                    }
+                ],
+                "history_text": {
+                    "scar": "m_c was scarred after foolishly trying to take on a c_n apprentice.",
+                    "death": "m_c died from injuries sustained after foolishly trying to take on a c_n apprentice."
+                },
+                "outsider_rep": -1
+            }
+        ],
+        "antag_fail_outcomes": [
+            {
+                "text": "The loner may not be causing any trouble <i>at the moment,</i> but that doesn't mean {PRONOUN/n_c:0/subject} won't in the future. p_l hisses out a warning, but the loner pays no attention, acting as though {PRONOUN/n_c:0/subject} {VERB/n_c:0/don't/doesn't} even <i>see</i> p_l!",
+                "exp": 0,
+                "weight": 20,
+                "new_cat": [
+                    ["meeting", "loner", "age:adolescent"]
+                ],
+                "relationships": [
+                    {
+                        "cats_to": ["n_c:0"],
+                        "cats_from": ["p_l"],
+                        "mutual": true,
+                        "values": ["dislike"],
+                        "amount": 10
+                    }
+                ],
+                "outsider_rep": -2
```

```suggestion
                "outsider_rep": -1
```
For `antag`, a fail should lower the rep less than a success, since the goal of antag is to decrease outsider rep (among other things). All the fails should probably be `-1` or `0` (unchanged is a totally valid outcome!)

I'd suggest going back through and reviewing the outsider rep changes with this in mind \:)